web/static/js/learning-maps/detail.js

```
@@ -0,0 +1,363 @@
+    function getNodeTypeLabel(type) {
+        switch (type) {
+            case 'tracker': return 'Progress Tracker';
+            case 'course': return 'Course';
+            case 'milestone': return 'Milestone';
+            default: return 'Node';
+        }
+    }
+
+    function shortenText(text, maxLength) {
+        if (text.length <= maxLength) return text;
+        return text.substring(0, maxLength - 3) + '...';
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)